modules/tables/table_11_Utkarsh.html

```diff
@@ -0,0 +1,75 @@
+<style>
+    *{
+        margin: 0;
+        padding: 0;
+        box-sizing: border-box;
+    }
+    .container{
+        background-image: url(./images_and_icons/bg.jpg);
+        height: 100vh;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        background-repeat: no-repeat;
+        background-size: cover;
+        overflow: hidden;
+    }
+    table{
+        background: white;
+        width: 60%;
+        height: 40%;
+    }
+    tr{
+        text-align: center;
+        /* all:unset; */
```

Comment:
Even though it's not mandatory, You could add comments to explain the next steps.